cmd/protoc-gen-cpp-tableau-loader/helper/helper.go

```diff
@@ -102,7 +102,8 @@ func AddMapKey(fd protoreflect.FieldDescriptor, keys []MapKey) []MapKey {
 	opts := fd.Options().(*descriptorpb.FieldOptions)
 	fdOpts := proto.GetExtension(opts, tableaupb.E_Field).(*tableaupb.FieldOptions)
 	name := strcase.ToSnake(strings.TrimSpace(fdOpts.GetKey()))
-	if name == "" {
+	name = escapeIdentifier(name)
+	if name == "" || name == "key" {
```

Comment:
find the protobuf map key field name in map value message

https://github.com/tableauio/loader/blob/9567d70a3080e6c831e9a15db752e37f18ab8ff9/cmd/protoc-gen-go-tableau-loader/check/check.go#L37-L45